concordium-consensus/src/Concordium/MultiVersion.hs

```diff
@@ -124,6 +124,12 @@ instance
     -- The information is used to count the number of finalized baked blocks exposed by a prometheus
     -- metric.
     handleFinalize _ lfbp bps = liftSkov $ do
+        -- Check for protocol update.
+        result <- checkForProtocolUpdate
+        let haveUnsupportedUpdate = case result of
+                CFPURUnsupportedPending -> True
```

Comment:
Shouldn't we return true for both unsupported variants?

Reply:
I have changed the design, so now it is calling a callback directly for signalling unsupported pending updates.
But the other case where it is "unsupported effective" will only be visible for a short time, since the node shuts down right after.